submissions/BlessingOkoro/php/fetch.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
You do not need this fetch.php file. What you are trying to do here can be done directly in `create.php` so you do not have to use Php's Session.